.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
           key: tox-${{ hashFiles('requirements/*.txt') }}-${{ hashFiles('pyproject.toml') }}-${{ hashFiles('.pre-commit-config.yaml') }}
 
       - name: Run tox
-        run: tox -e lint,py,coverage-report,typing  # run tox using Python in path
```

Comment:
Is there a reason you need to remove the `typing` step?

Reply:
As far as I can tell it never actually existed.  The version of tox in this repo is unpinned.  I think it got upgraded at some point, and I'm guessing this changed from a warning to an error.

Tox was returning
```
ROOT: HandledError| provided environments not found in configuration file:
typing
```
and from what I can see there is indeed no `[testenv:typing]` environment defined in pyproject.toml.  Other projects using the same template as this one do include such an environment, see e.g. https://github.com/lsst-sqre/datalinker/blob/2ad0bfe0aca7ddf463c9f8a91c29ddd7533c320a/tox.ini#L25

Mypy is included in the pre-commit config, and pre-commit is getting run on all files in the `lint` step.  So mypy is still running, just through a separate tool.  https://github.com/lsst-sqre/exposurelog/actions/runs/8145000061/job/22260306926#step:6:41

Reply:
Oh you are right, I missed we were using the Mypy tool. Thanks for the explanation @dhirving  :pray: !